rust/kcl-lib/src/lint/checks/camel_case.rs

```diff
@@ -23,29 +24,42 @@ https://en.wikipedia.org/wiki/Camel_case
 "
 );
 
-fn lint_lower_camel_case_var(decl: &VariableDeclarator) -> Result<Vec<Discovered>> {
+fn lint_lower_camel_case_var(decl: &VariableDeclarator, prog: &AstNode<Program>) -> Result<Vec<Discovered>> {
     let mut findings = vec![];
     let ident = &decl.id;
     let name = &ident.name;
 
     if !name.is_case(convert_case::Case::Camel) {
+        // Get what it should be.
+        let new_name = name.to_case(convert_case::Case::Camel);
+
+        let mut prog = prog.clone();
+        prog.rename_symbol(&new_name, ident.start);
+        let recast = prog.recast(&Default::default(), 0);
+
+        let suggestion = Suggestion {
+            title: format!("rename '{}' to '{}'", name, new_name),
+            insert: recast,
+            source_range: prog.as_source_range(),
+        };
         findings.push(Z0001.at(
             format!("found '{}'", name),
             SourceRange::new(ident.start, ident.end, ident.module_id),
-            None,
+            Some(suggestion.clone()),
         ));
         return Ok(findings);
     }
 
     Ok(findings)
 }
 
-fn lint_lower_camel_case_property(decl: &ObjectProperty) -> Result<Vec<Discovered>> {
+fn lint_lower_camel_case_property(decl: &ObjectProperty, _prog: &AstNode<Program>) -> Result<Vec<Discovered>> {
```

Comment:
Is this meant to be left in?

Reply:
yeah it impliments a trait and eventually I will hook it thru once i can rename property keys